src/dsql/DdlNodes.epp

```diff
@@ -11996,9 +11996,14 @@ void AlterDatabaseNode::execute(thread_db* tdbb, DsqlCompilerScratch* dsqlScratc
 
 			// msg 297: Concurrent ALTER DATABASE is not supported
 			Arg::PrivateDyn status(297);
+			string trans_num_str;
 
 			if (conflict_trans)
-				status << Arg::Gds(isc_concurrent_transaction) << Arg::Num(conflict_trans);
+			{
+				// Cannot use Arg::Num here because transaction number is 64-bit signed integer
```

Comment:
Here and later - is it good idea to use SQUADFORMAT (which is for signed numbers) to print values of TraNumber datatype which is:
typedef FB_UINT64 TraNumber;
(i.e. unsigned)?

Reply:
In FB3 I see this:
typedef SINT64 TraNumber;

Reply:
Sorry, I really looked at master branch.

Reply:
Using of macros from stdint.h would be more standard-friendly.